Testscripts/Windows/VM-HOT-RESIZE.ps1

```diff
@@ -20,8 +20,28 @@ function Main {
 
         $loadBalanceName = (Get-AzureRmLoadBalancer -ResourceGroupName $AllVMData.ResourceGroupName).Name
         $VirtualMachine = Get-AzureRmVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName
-        for ( $i = 0; $i -le 30; $i++ ) {
+        foreach ($param in $currentTestData.TestParameters.param) {
+            if ($param -match "TestMode") {
+                $testMode = $param.Replace("TestMode=","").Replace('"',"")
+            }
+        }
+        if ($testMode -eq "economy") {
+            Write-LogInfo "The test mode is economy mode"
+            $toalTestTimes = 10
+        } else {
+            $toalTestTimes = 30
+        }
+
+        for ( $i = 0; $i -le $toalTestTimes; $i++ ) {
```

Comment:
typo in total.
remove spaces in the ( ) syntax, no need to (_space_$i and at the end of it

Reply:
the typo still remains in all places, please fix.

Reply:
It's fixed. Please check the updated code.

Reply:
@chvalean Please check against latest update code.

Reply:
> @chvalean Please check against latest update code.

I still see the word "toal".
I might be misunderstanding the word?

Reply:
My fault. It's fixed.